ts/features/payments/bizEventsTransaction/screens/PaymentsTransactionBizEventsPreviewScreen.tsx

```diff
@@ -47,26 +47,11 @@ const PaymentsTransactionBizEventsPreviewScreen = () => {
     }
     try {
       await Share.open({
-        activityItemSources: [
-          {
-            item: {
-              default: {
-                content: `${I18n.t(
-                  "features.payments.transactions.receipt.title"
-                )}.pdf`,
-                type: "url"
-              }
-            },
-            placeholderItem: {
-              content: `${I18n.t(
-                "features.payments.transactions.receipt.title"
-              )}.pdf`,
-              type: "url"
-            }
-          }
-        ],
         type: "application/pdf",
-        url: `${RECEIPT_DOCUMENT_TYPE_PREFIX}${transactionReceiptFile}`
+        url: `${RECEIPT_DOCUMENT_TYPE_PREFIX}${transactionReceiptFile}`,
+        filename: `${I18n.t(
+          "features.payments.transactions.receipt.title"
+        )}.pdf`
       });
     } catch (err) {
       // Don't do anything if the user cancels the share action
```

Comment:
Should we handle the case where the sharing action actually fails?
With the "failOnCancel: false" parameter we can avoid triggering errors when the user cancels the sharing action.

Reply:
Thanks for the suggestion 🙏! I added the property `failOnCancel: false` in the following commit: 0ab0a50ca3f795a8264455e8a16bd69ec5fe5f23